Ieva stonkaite/feature/t 9 feature branch for pom refactoring

IevaStonkaite opened the pull request:
First task - login.js and loginPage.js
Second task - t2.js and t2Page.js

Commit:
Changed Demo User to Team Lead
Added date check

Commit:
Merge branch 'Ieva_Stakneviciute/features/TC1-verify_current_date' of https://github.com/nikolaj-tolkaciov/sourcery-cypress into ieva_stonkaite/feature/T1-validate_todays_date

# Conflicts:
#	cypress/integration/login.js

Commit:
T2 finished task

Commit:
Merge branch 'ieva_stonkaite/feature/T2-verify_all_user_logs' of https://github.com/nikolaj-tolkaciov/sourcery-cypress into ieva_stonkaite/feature/T2-verify_all_user_logs

Commit:
TC2 conflict resolved

Commit:
T2 - lithuanian comments removed

Commit:
T1 and half of T2 refactoring

Commit:
Tasks T1 and T2 refactored

Commit:
T1 finished, T2 started in separate files

Commit:
Task1 (login.js) and task 2 (t2.js) refactored

Commit:
Deleted unnecessary file

Commit:
Merge pull request #146 from nikolaj-tolkaciov/ieva_stonkaite/feature/T2-verify_all_user_logs

Ieva stonkaite/feature/t2 verify all user logs

linukkis reviewed (commented):
Please adjust your branch so that the pull request points to your local develop at `ieva_stonkaite/develop` ;)

Commit:
Created a file called "timeLoggingPage" and "loginPage" as advised;
Deleted repeated methods;
Deleted empty rows.

Commit:
Created a new file "common.js" with functions to get page title and user title;
Moved task1 and task2 into one file (login.js), instead two files, because they check were checking login functionality.

Commit:
Added more methods to common.js, deleted timeLoggingPage.js

Commit:
Additional fix for the urls

Commit:
Merge branch 'ieva_stonkaite/develop' into ieva_stonkaite/feature/T-9_feature_branch_for_POM_refactoring

# Conflicts:
#	cypress/integration/login.js

Commit:
fixed misleading function name, deleted pointless function and dubblicated functions
